CMakeLists.txt

```diff
@@ -392,6 +390,8 @@ else ()
   set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} "${ENSMALLEN_INCLUDE_DIR}")
 endif ()
 
+find_package(cereal 1.3.0)
```

Comment:
Do we need to add, e.g., `CEREAL_INCLUDE_DIR` to `MLPACK_INCLUDE_DIRS`?

Reply:
Actually, I am not sure, I did not have time to verify if there is a specific cmake file for find_package in cereal side.